### PR TITLE
kubeadm-install: modify the note about kubelet cgroup drivers

### PR DESCRIPTION
**Pull request by neolit123**
previous PR by @erickhun:
https://github.com/kubernetes/website/pull/9410

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews 
/cc @Bradamant3 

/kind cleanup

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubeadm.md [hunk 1]
```diff
@@ -234,23 +234,18 @@ kubeadm to tell it what to do.
...
-
-```bash
-sed -i "s/cgroup-driver=systemd/cgroup-driver=cgroupfs/g" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-```
+If you are using a different CRI, you have to modify the the drop-in file
+`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`, by adding
+`--cgroup-driver=<driver-name>`.
 
-Otherwise, you will need to open the systemd file and add the flag to an existing environment line.
+Please mind, that you **only** have to do that if the cgroup driver of your CRI
```
Comment (timothysc) [inline content/en/docs/tasks/tools/install-kubeadm.md]: This sentence is odd. 

I'm going to defer to @kad  to lgtm 

/approve 
/cc @kad

Comment (neolit123) [reply content/en/docs/tasks/tools/install-kubeadm.md]: for reference:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1beta1/defaults.go#L125-L127

```
	if obj.CgroupDriver == "" {
		obj.CgroupDriver = "cgroupfs"
	}
```

File: content/en/docs/tasks/tools/install-kubeadm.md [hunk 2]
```diff
@@ -234,23 +234,18 @@ kubeadm to tell it what to do.
 
 ## Configure cgroup driver used by kubelet on Master Node
 
-Make sure that the cgroup driver used by kubelet is the same as the one used by Docker. Verify that your Docker cgroup driver matches the kubelet config:
+When using Docker, kubeadm will detect the cgroup driver for the kubelet
```
Comment (chenopis) [inline content/en/docs/tasks/tools/install-kubeadm.md]: Suggested edit:
"When using Docker, kubeadm will automatically detect the cgroup driver for the kubelet
 and set it in the `/var/lib/kubelet/kubeadm-flags.env` file
during runtime."

**Comment by chenopis** [comment]
@neolit123 Please address the conflicts and any remaining feedback. Thx

File: content/en/docs/setup/independent/install-kubeadm.md [hunk 3]
```diff
@@ -234,23 +234,17 @@ kubeadm to tell it what to do.
...
-```
-
-If the Docker cgroup driver and the kubelet config don't match, change the kubelet config to match the Docker cgroup driver. The
-flag you need to change is `--cgroup-driver`. If it's already set, you can update like so:
-
-```bash
-sed -i "s/cgroup-driver=systemd/cgroup-driver=cgroupfs/g" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-```
+If you are using a different CRI, you have to modify the the drop-in file
+`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`, by adding
```
Comment (kad) [inline content/en/docs/setup/independent/install-kubeadm.md]: It is not good to modify configuration file that comes from package. Better to mention on how to add user's drop-in with override of cgroups-driver argument.

**Comment by neolit123** [comment]
updated with the latest change request by @kad.

**Pull request merged**